src/vulkan_include.hpp

```diff
@@ -1,15 +1,15 @@
 #ifndef VULKAN_INCLUDE_HPP_INCLUDED
 #define VULKAN_INCLUDE_HPP_INCLUDED
 
-#include <assert.h> //vulkan/vk_dispatch_table_helper.h needs this
+#include <assert.h> // vk_dispatch_table_helper.h needs this
 
 #define VK_NO_PROTOTYPES
 
 #pragma GCC system_header
 #include "vulkan/vulkan.h"
 #include "vulkan/vk_layer.h"
-#include "vulkan/vk_layer_dispatch_table.h"
-#include "vulkan/vk_dispatch_table_helper.h"
+#include "vk_layer_dispatch_table.h"
```

Comment:
This doesn't build for me.

Reply:
Ah wow I just realized that Debian doesn't have the prefix while Fedora has it.

Fun fact: before 1.2.159, the headers actually don't go into the `vulkan/` dir, that was introduced with this commit: https://github.com/KhronosGroup/Vulkan-ValidationLayers/commit/ddeed65b37f58f7ef2efaae92ea23ab99dbc6929

The problem is: that will break the Wrap, so I'll guess this is stalled until I figured something out.

Reply:
I guess we could do something like this even if it's really ugly
```c
#if defined __has_include
#if __has_include ("vk_layer_dispatch_table.h") && __has_include("vk_dispatch_table_helper.h")
#include "vk_layer_dispatch_table.h"
#include "vk_dispatch_table_helper.h"
#else
#include "vulkan/vk_layer_dispatch_table.h"
#include "vulkan/vk_dispatch_table_helper.h"
#endif
#else
#include "vulkan/vk_layer_dispatch_table.h"
#include "vulkan/vk_dispatch_table_helper.h"
#endif
```

Reply:
Tbh it's not an urgent matter so I would wait until the Vulkan people decide something.

I opened a WIP PR here: https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/2331
So let's wait how they respond.